src/test/java/org/springframework/data/hazelcast/repository/query/HazelcastSortAccessorTest.java

```diff
@@ -0,0 +1,174 @@
+package org.springframework.data.hazelcast.repository.query;
+
+import com.hazelcast.query.Predicate;
+import org.junit.Test;
+import org.springframework.data.domain.Sort;
+import org.springframework.data.history.RevisionSort;
+import org.springframework.data.keyvalue.core.query.KeyValueQuery;
+import org.springframework.util.ObjectUtils;
+
+import java.io.Serializable;
+import java.util.*;
+
+import static org.junit.Assert.*;
+
+public class HazelcastSortAccessorTest {
+
+    @Test()
```

Comment:
wouldn't this be better `@Test(expected = UnsupportedOperationException.class)`? instead of adding try and catch block?

Reply:
The idea was to have 'given/when/then' structure to testcases. ( as per previous review )

Reply:
You can just add the comment at the end:

```
// Then expect UnsupportedOperationException
```

Reply:
@dineshbhagat , @mtopolnik: I have addressed your review comments. 
So, this provides a cleaner structure in testcases, with the annotation.

Reply:
👍